jukebox/src/main/webapp/js/script.js

```diff
@@ -131,10 +137,14 @@ function createRoom() {
             db.collection("rooms").doc(pinCode).set( {
                 name: document.getElementById("group_name").value,
             })
-            db.collection("rooms").doc(pinCode).collection("users").doc(storedUid).set( {
-                isHost: true
+            .then(function (groupCreatedRef) {
+                db.collection("rooms").doc(pinCode).collection("users").doc(storedUid).set( {
+                    isHost: true
```

Comment:
I'm a bit unsure about this line - what does "isHost" represent in this context?